### ISSUE TITLE
cmd/tsuru: support interactive mode in tsuru app-run

### ISSUE THREAD

**Issue by fsouza** [tsuru/tsuru#851]
``tsuru run`` should have an interactive mode, where user can 

Something like:

```
% tsuru app-run -i python manage.py syncdb
Are you sure? y
```

It will run only once, even when the application has more than one unit (i.e.: ``-i`` implies ``-o``).

**Comment by fsouza** [comment]
This has been superseded by #1090.
